views/pages/admin/image-store.pug

```diff
@@ -12,7 +12,7 @@ block body
     div(class="actions")
         input(type="text" name="image-search-input" placeholder="Search for images...")
         form(method="POST" action="/api/image/upload" enctype="multipart/form-data")
-            input(type="file" name="image" required multiple accept="image/*")
+            input(type="file" name="images" required multiple accept="image/*")
```

Comment:
Uploading a .webp file is allowed by the selector but not allowed by the server.

- If we want to allow it, can you add it to the list of acce[ted file types?
- If we don't want to allow it, can there be an error toast that indicates a failure to upload, preferably with a list of accepted file types in the error message?

I can't think of a reason to disallow .webp but let me know if there is one!